README.md

```diff
@@ -1,13 +1,20 @@
 # Calculator App 
 
-This project builds a web-based calculator for a fictional company named Math-magicians. The business requirement is to build a simple product that would assist the company with their calculations.
+This project builds a web-based calculator for a fictional company named Math-magicians. The business requirement is to build a simple product that would assist the company with their calculations. The Math Magicians website has three pages, the home page, the calculator page and the quote page. 
 
-![Screenshot](src/App_screenshot.png)
+## Home Page
+![Screenshot](src/Home.png)
+
+## Caclculator Page
+![Screenshot](src/Calculator.png)
+
+## Quote Page 
+![Screenshot](src/Quote.png)
 
 ### Components 
 The project applies React's best practices and puts use of functional and class components. 
-It has Button, ButtonPanel and Display components which renders UI elements. Finally the App component renders ButtonPanel and Display components. 
-The App has state attributes, 
+It has Button, ButtonPanel and Display components which renders UI elements. Finally the CalculatorPage component renders ButtonPanel and Display components. 
+The CalculatorPage component has hooks.
 
```

Comment:
README file not detailed enough for the newbie(non-technical individual), You should add steps to git clone your repo, add necessary installation to setup project locally and you need to update your live demo link.

Reply:
Thanks, @bellom. Now I made the requested changes.